src/content/protocols/uniswap-v3.md

```diff
@@ -0,0 +1,117 @@
+---
+protocol: "Uniswap-V3"
+website: "https://blog.uniswap.org/uniswap-v3"
+x: "https://x.com/uniswap"
+github: "https://github.com/Uniswap/v3-periphery/"
+defillama_slug: "uniswap-v3"
+chain: "Ethereum Mainnet"
+stage: 1
+risks: ["L", "M", "L", "M", "L"]
+author: ["Emilien"]
+submission_date: "2024-11-12"
+publish_date: "2024-12-14"
+acknowledge_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+Uniswap v3 is an AMM that builds upon Uniswap v2 by introducing a concentrated liquidity model, providing liquidity providers with granular control over capital allocation. Unlike v2, where liquidity is distributed uniformly across all price ranges, v3 allows LPs to specify custom price ranges in which their liquidity is active. This approach significantly improves capital efficiency, as LPs can concentrate their assets in high-demand price ranges, earning fees only within those specified ranges.
+Uniswap v3 also introduces multiple fee tiers (0.01%, 0.05%, 0.3%, and 1%) to support different asset volatility profiles, allowing LPs to adjust their fee preferences based on expected risk and return. Additionally, it incorporates "range orders," which effectively turn liquidity positions into limit orders, further enhancing LP strategy flexibility.
+The protocol is deployed across multiple chains enabling a wide range of use cases across decentralized finance (DeFi) applications.
+
+# Overview
+
+## Chain
+
+Uniswap is deployed on Ethereum mainnet, Arbitrum, Optimism, Polygon, Base, BNB,
+Avalanche C-Chain, CELO, Blast, ZKsync, Zora, and WorldChain. This analysis is
+based on the contracts deployed on Ethereum mainnet.
+
+> Chain score: L
+
+## Upgradeability
+
+Uniswap pools are permissionless with anyone being able to swap. Nonetheless the Uniswap DAO
+(UNI token holders) can change parameters such as fees through the GorvernorBravoDelegator contract.
+Apart from the fees set by the governance, the protocol's contracts are immutable. No party is able to
+pause, reverse trade execution, or otherwise change the behavior of the protocol.
+
+Note that a TransparentProxy with the DAO as admin is used for the NonFungibleTokenPositionDescriptor, which is used for token descriptions.
+This should not impact the safety of the protocol.
+
+> Upgradeabillity score: M
+
+## Autonomy
+
+There are no particular dependencies for the Uniswap protocol.
+
+## Exit Window
+
+As the contracts are immutable the users can always withdraw their funds, but parameters such as protocol
+fees can be changed by the DAO. Propositions can be voted on over the duration of a week, and if a
+proposal is approved it can take effect between 2 and 30 days later.
+
+> Exit score: M
+
+## Accessibility
+
+Although there is only a unique official uniswap frontend, Uniswap Labs offers an [open source interface](https://github.com/Uniswap/interface) for all Uniswap protocols. The web app is also hosted on IPFS.
```

Comment:
please add this link: https://github.com/Uniswap/interface/releases

(here are the latest deployments on IPFS)

Reply:
and also list this link: https://blog.uniswap.org/uniswap-interface-ipfs